File: fizz-buzz.pl
sub fizzbuzz {
    if ($_[0] % 3 == 0 && $_[0] % 5 == 0) { return "Fizz Buzz"; }
    if ($_[0] % 3 == 0) { return "Fizz"; }
    if ($_[0] % 5 == 0) { return "Buzz"; }
    return $_[0];
}

my $n = <STDIN>;
for (my $i = 1; $i <= $n; $i++) { print fizzbuzz($i) . "\n"; }

File: fizz-buzz.js
const n = require("fs").readFileSync("/dev/stdin", "utf8");

function fizzbuzz(i) {
    if (i % 3 == 0 && i % 5 == 0) return "Fizz Buzz";
    if (i % 3 == 0) return "Fizz";
    if (i % 5 == 0) return "Buzz";
    return i;
}

for (var i = 1; i <= n; i++) console.log(fizzbuzz(i));